Fixed errors on coming from downloading main branch

SkylarPlayz348 opened the pull request:
fixed an issue when dowloading the main branch and it causing error when running the index.js and also got rid of the random string of numbers that was meant to be gone from my last pr

SkylarPlayz348 committed:
Fixed errors on coming from downloading main branch

fixed an issue when dowloading the main branch and it causing error when running the index.js

braxtongpoll commented:
The millisecond string was added to ensure no conflict. If it isn't there and lets say you do 2 `program.js` files on the same day, one of the will not backup, as the backups will have the same name.

SkylarPlayz348 committed:
more issues fixed and reverted back

I'll be back with a solution to what i want to do but if you see i want to do something about that `Date.now()`

SkylarPlayz348 committed:
got rid of testing code

SkylarPlayz348 commented:
fixed plus fixed some fatal errors that i came across

SkylarPlayz348 committed:
just to clean it up
